politeiawww/cmd/politeiavoter/politeiavoter.go

```diff
@@ -646,8 +723,11 @@ func (c *ctx) inventory() error {
 			continue
 		}
 
+		name, _ := names[t]
+
 		// Display vote bits
 		fmt.Printf("Vote: %v\n", dr.Vote.Params.Token)
+		fmt.Printf("  Proposal        : %v\n", name)
```

Comment:
You can do this inline to make it cleaner.

```suggestion
		fmt.Printf("  Proposal        : %v\n", names[t])
```